src/state/application/updater.ts

```diff
@@ -79,19 +77,20 @@ export default function Updater(): null {
     )
   }, [dispatch, debouncedState.chainId])
 
+  const implements3085 = useAppSelector((state) => state.application.implements3085)
+
   useEffect(() => {
-    const isCbWalletDappBrowser = window?.ethereum?.isCoinbaseWallet
-    const isWalletlink =
-      connector instanceof WalletLinkConnector || (connector instanceof InjectedConnector && window.walletLinkExtension)
-    const isCbWallet = isCbWalletDappBrowser || isWalletlink
-    const isMetamaskOrCbWallet = library?.provider?.isMetaMask || isCbWallet
-    if (!account || !library?.provider?.request || !isMetamaskOrCbWallet) {
-      return
+    if (account && library?.provider?.request) {
```

Comment:
doesn't make as much sense to check for specific connectors now that we know more than metamask supports the eip